resources/conf/config.toml

```diff
@@ -56,6 +56,7 @@
 [[enforcer.security.tokenService]]
   name = "APIM Publisher"
   issuer = "https://localhost:9443/publisher"
+  validateSubscription = true
```

Comment:
shall we add this for k8s artficat (without apim) as well?

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/2526/commits/3f330389b5eb4611d89039401b1b4ad3c0ef265e